client/ayon_maya/plugins/load/load_layout.py

```diff
@@ -0,0 +1,231 @@
+        return asset
+
+    def _process(self, filepath, options, loaded_options=None):
+
+        with open(filepath, "r") as fp:
+            data = json.load(fp)
+
+        all_loaders = discover_loader_plugins()
+
+        if not loaded_options:
+            loaded_options = []
+
+        repre_entities_by_version_id = self._get_repre_entities_by_version_id(
+            data
+        )
+        for element in data:
+            repre_id = None
+            repr_format = None
+            version_id = element.get("version")
+            if version_id:
+                repre_entities = repre_entities_by_version_id[version_id]
+                if not repre_entities:
+                    self.log.error(
+                        f"No valid representation found for version"
+                        f" {version_id}")
+                    continue
+                repre_entity = repre_entities[0]
+                repre_id = repre_entity["id"]
+                repr_format = repre_entity["name"]
+
+
+            # If reference is None, this element is skipped, as it cannot be
+            # imported in Maya
+            if not repre_id:
+                continue
+
+            instance_name = element.get('asset_name')
+            extension = instance_name.split("_")[-1]
+            instance_name = instance_name.replace(f"_{extension}", "")
+            containers = [
+                con for con in cmds.ls(f"{instance_name}*")
+                if con.endswith("_CON")
+            ]
+            if not containers:
+                if repre_id not in loaded_options:
+                    loaded_options.append(repre_id)
+
+                    product_type = element.get("product_type")
+                    if product_type is None:
+                        product_type = element.get("family")
+                    loaders = loaders_from_representation(
+                        all_loaders, repre_id)
+
+                    loader = None
+
+                    if repr_format:
+                        loader = self._get_loader(loaders, product_type)
+
+                    if not loader:
+                        self.log.error(
+                            f"No valid loader found for {repre_id}")
+                        continue
+
+                    options = {
+                        # "asset_dir": asset_dir
+                    }
+                    load_container(
+                        loader,
+                        repre_id,
+                        namespace=instance_name,
+                        options=options
+                    )
+                instances = [
+                    item for item in data
+                    if ((item.get('version') and
+                        item.get('version') == element.get('version')))]
+
+                for instance in instances:
+                    transform = instance.get('transform')
+                    instance_name = instance.get('asset_name')
```

Comment:
```suggestion
                    transform = instance["transform"]
                    instance_name = instance["asset_name"]
```
Logic below would fail anyway if these were None, so it seems like we require these to exist.